node/service/Cargo.toml

```diff
@@ -151,3 +151,6 @@ runtime-benchmarks = [
 	"moonbeam-runtime/runtime-benchmarks",
 	"moonbase-runtime/runtime-benchmarks",
 ]
+try-runtime = [
+	"moonbase-runtime/try-runtime",
+]
```

Comment:
So it's only on moonbase runtime right now right? I'm fine with that. Probably better to start there.

Reply:
That's only because it was the first I tried :) I can add to the others before we merge, I'm fine either way.

Reply:
I think it makes sense to only do this one to start.